refactor/__main__.py

```diff
@@ -41,8 +41,7 @@ def main() -> int:
     )
 
     options = parser.parse_args()
-    if options.refactor_file:
-        refactor_file = options.refactor_dir
+    if refactor_file := options.refactor_file:
         if refactor_file.exists():
```

Comment:
I think we should also fix this to `not refactor_file.exists()` to fix the use case all together.  I think I've always used the default files, so didn't even try specifying a custom one.